lib/src/rpc.mli

```diff
@@ -1,6 +1,7 @@
+open Base
 open Async.Rpc
 
-val block_get : (string, (string * bool) option) Rpc.t
+val block_get : (string, (string, bool) Either.t, unit) Pipe_rpc.t
```

Comment:
It would be great to add a comment explaining the type signature. What do those `string`s and `bool` stand for?

Looking at the implementation, this is used to express the fact that we want to respond with a single executable bit plus a stream of strings (file content). 

The current encoding is not great since we carry the cost of `First`/`Second` constructor in all messages, and we can probably commit it with a big TODO, but we also need some comments to explain the problem. (The problem being the lack of a way to express what we need using `Pipe_rpc` interface.)